_raml/apps/products/products.raml

```diff
@@ -130,23 +130,51 @@ post:
     responses:
       204:
 
-  /crossselling:
-    displayName: Product crossselling
-    get:
-      is: [ locale, currency ]
-      queryParameters:
-        includeInvisible:
-          description: Includes invisible products. Only available with `products_write` authorisation.
-          type: boolean
-          required: false
-      description: Returns information on the crossselling of products. *epagesNow
-      securedBy: [products_read]
-      responses:
-        200:
-          body:
-            application/json:
-              example: !include examples/get_product_productid_crossselling_response.json
-              schema: !include schema/get_product_productid_crossselling.schema.json
+/crossselling:
+  displayName: Product crossselling
+  get:
+    is: [ locale, currency ]
+    queryParameters:
+      includeInvisible:
+        description: Includes invisible products. Only available with `products_write` authorisation.
+        type: boolean
+        required: false
+    description: Returns information on the crossselling of products. *epagesNow
```

Comment:
To match the description of the two new requests, I would suggest to adapt the wording as follows:
```suggestion
    description: Returns information on the cross-selling assignments of a single product. *epagesNow
```
Because an ID is used in the request, I guess it refers to a single product instead of several ones.

Reply:
This PR does not change anything in relation to the GET request, but I've updated this anyway